features/step_definitions/common.ts

```diff
@@ -13,21 +14,51 @@ Given(
 );
 
 Given(
-  "The app {string} with {string} has no records",
-  function (appId: string, apiToken: string) {
-    const command = `record delete --app ${appId} --base-url $$TEST_KINTONE_BASE_URL --api-token ${apiToken} --yes`;
-    this.execCliKintoneSync(command);
-    if (this.response.status !== 0) {
-      throw new Error(`Resetting app failed. Error: \n${this.response.stderr}`);
-    }
+  "Load app ID of app {string} as env var: {string}",
+  function (appKey: string, destEnvVar: string) {
+    const credential = this.getCredentialByAppKey(appKey);
+    this.env = { [destEnvVar]: credential.appId, ...this.env };
+  },
+);
+
+Given(
+  "Load app token of app {string} with permission {string} as env var: {string}",
+  function (appKey: string, permission: Permission, destEnvVar: string) {
+    const apiToken = this.getAPITokenByAppAndPermission(appKey, {
+      [permission]: true,
+    });
+    this.env = { [destEnvVar]: apiToken, ...this.env };
   },
 );
 
+Given(
+  "Load app token of app {string} without permission {string} as env var: {string}",
+  function (appKey: string, permission: Permission, destEnvVar: string) {
+    const apiToken = this.getAPITokenByAppAndPermission(appKey, {
+      [permission]: false,
+    });
+    this.env = { [destEnvVar]: apiToken, ...this.env };
+  },
+);
```

Comment:
How about re-design like the following?
- Update step `Load app token of app {string} with permission {string} as env var: {string}` to accept multiple permissions
  - comma separated? (`"view, edit"`, `"delete"`)
  - The step will return the token that EXACTLY have the specified permissions.
- Remove `Load app token of app {string} without permission {string} as env var: {string}`

Reply:
@mshrtsr 

I think It's good. Let's try it.

For test cases, such as "no privilege to add record", we need to clearly describe the scenario summary to avoid confusion. For example: `Scenario: CliKintoneTest-28 Should return the error message when importing records with API Token that have no privilege to add record`

Besides, I think we could use the definition step below to make it more clear.
`Load app token of app {string} with exactly permissions {string} as env var: {string}`